tests/test_sensitivity_analysis.py

```diff
@@ -151,7 +151,12 @@ def sobol_results_1d(model_1d):
 # # test conversion to DataFrame --------------------------------------------------
 @pytest.mark.filterwarnings("ignore::FutureWarning")
 def test_sobol_results_to_df(sobol_results_1d):
```

Comment:
Perhaps the case when `problem=None` could also be tested with `pytest.parameterize`?
```suggestion
@pytest.mark.parametrize(
    "problem, expected_names",
    [
        (
            {
                "num_vars": 2,
                "names": ["c", "v0"],
                "bounds": [(-5.0, 1.0), (0.0, 1000.0)],
            },
            ["c", "v0", "c-v0"],
        ),
        (None, ["X1", "X2", "X1-X2"]),
    ],
)
def test_sobol_results_to_df(sobol_results_1d, problem, expected_names):
```